templates/taxi/car_detail.html

```diff
@@ -1,22 +1,43 @@
 {% extends "base.html" %}
 
```

Comment:
According to the code style guide, you should use 2 whitespaces indentations in your `.html` files. Please, consider this for future code writing.